### ISSUE TITLE
Disable floating stemcells before tile upgrade

### ISSUE THREAD

**Issue by benfab** [pivotal-cf/pcf-pipelines#151]
Hello, 

Before starting a tile upgrade in a large scale environment, I would like to make sure the floating stemcell feature is disabled. It would be nice to have this checked as part of the upgrade-tile pipeline.

See: 
https://docs.pivotal.io/pivotalcf/1-7/customizing/understanding-stemcells.html
https://discuss.pivotal.io/hc/en-us/articles/224704307-How-to-disable-floating-stemcells-in-Pivotal-Cloud-Foundry

Thanks!

**Comment by ryanpei** [comment]
@benfab thanks for using our pipelines! I'm not familiar with the reason(s) why anyone wants to disable floating stemcells, although it does seem like this is not uncommon if Pivotal has an article explaining how to disable this. Can you please let me know what the reason(s)?

**Comment by benfab** [comment]
When activated during a tile upgrade requiring a new stemcell, the floating stemcells feature will force all VMs from all deployments to be recreated with the new stemcell. In very large scale setup, this behavior will cause a single tile update to take significantly longer wait time, which in production needs to be carefully handled.

**Comment by RMeharg** [comment]
Hi Ryan,

We also have a requirement to disable floating stemcells, at least for running production upgrades, it would be great to see this as a feature of pcf-pipelines.

**Comment by abbyachau** [comment]
Hi @benfab, thank you for opening this issue - and sorry it's taken so long to add further remarks. I plan on closing this issue because we do not have plans to add this to our backlog - and also Ops Manager has plans to allow you to opt out of using floating stemcells thereby allowing you to control when a tile's stemcell is upgraded. Please do not hesitate to get in touch should you have any questions.
